board/drivers/can.h

```diff
@@ -326,9 +366,9 @@ void can_rx(uint8_t can_number) {
 
     safety_rx_hook(&to_push);
 
-    set_led(LED_BLUE, 1);
+    current_board->set_led(LED_BLUE, true);
     can_send_errs += !can_push(&can_rx_q, &to_push);
-
+    
```

Comment:
spaces

Reply:
Not sure what you mean here. Might already be fixed?

Reply:
just 4 trailing spaces on the empty line.